Source/items.cpp

```diff
@@ -490,11 +490,11 @@ void CalcSelfItems(Player &player)
 		for (int i = 0; i < NUM_INVLOC; i++, pi++) {
 			if (!pi->isEmpty() && pi->_iStatFlag) {
 				bool sf = true;
-				if (sa + player._pBaseStr < pi->_iMinStr)
+				if (pi->_iMinStr != 0 && sa + player._pBaseStr < pi->_iMinStr)
 					sf = false;
-				if (ma + player._pBaseMag < pi->_iMinMag)
+				if (pi->_iMinMag != 0 && ma + player._pBaseMag < pi->_iMinMag)
 					sf = false;
-				if (da + player._pBaseDex < pi->_iMinDex)
+				if (pi->_iMinDex != 0 && da + player._pBaseDex < pi->_iMinDex)

```

Comment:
While it wasn't part of Diablo 1, it would be cool to have Vitality as a stat requirement for items in mods.

Reply:
but the compatibility ... xD